lib/util.js

```diff
@@ -92,10 +105,25 @@ function escapeStyleCode(code) {
 /**
  * @param {string | string[]} format
  * @param {string} text
+ * @param {object} [options={}]
+ * @param {boolean} [options.validateStream=true] - Whether to validate the stream.
+ * @param {Stream} [options.stream=process.stdout] - The output stream to write the styled text to.
```

Comment:
The function isn't writing the output to the stream, so this may be misleading.

```suggestion
  * @param {Stream} [options.stream=process.stdout] - The stream to use for validation
```